core-backend/wasmtime/Cargo.toml

```diff
@@ -11,6 +11,7 @@ gear-core-errors = { path = "../../core-errors", features = ["codec"] }
 gear-backend-common = { path = "../common" }
 
 wasmtime = { version = "0.35.1", default-features = false, features = ["parallel-compilation", "cranelift"] }
+anyhow = "1.0.57"
```

Comment:
should we disable default features?

Reply:
there is the `std` one

Reply:
https://github.com/gear-tech/gear/blob/3009d0e2fe986abf11511975e12056d1d0b6f1ee/core-backend/wasmtime/src/lib.rs#L21
Isn't it?